src/main/java/com/github/tomakehurst/wiremock/matching/EqualToUrlEncodedFormPattern.java

```diff
@@ -51,24 +64,35 @@ public double getDistance() {
         };
     }
 
-    private Map<String, String> parse(String urlEncodedForm) {
+    private Map<String, String> parse(String urlEncodedForm, List<StringValuePattern> formParametersToMatch) {
```

Comment:
Parse is used on recording (to save only the params we want) and playback (to try and match the outgoing request using only the params that pass the filters it was recorded with). This is also safer than ignoring all extra params: a real request might have a param that's missing from the recorded request but matches its filter. In that case we probably care about that param and don't want to assume the recording that lacks it is a match for our request.